packages/Select/src/SelectHeader.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import cx from 'classnames';
+
+import './select-header.scss';
+
+import ChevronUp from '@igloo-ui/icons/dist/TableChevronSelectedUp';
+import ChevronDown from '@igloo-ui/icons/dist/TableChevronSelectedDown';
+
+export interface SelectHeaderProps extends React.ComponentProps<'div'> {
+  /** Default value displayed in the Select. */
+  children?: React.ReactNode;
+  /** True if the Select is disabled. */
+  disabled?: boolean;
+  /** True if the option list is displayed. */
+  isOpen?: boolean;
+}
+
+const SelectHeader: React.FunctionComponent<SelectHeaderProps> = (
+  props: SelectHeaderProps
+) => {
+  const { children, disabled = false, isOpen = false, ...rest } = props;
+
+  const selectHeaderClasses = cx('ids-select-header', {
+    'ids-select-header--disabled': disabled,
+    'ids-select-header--active': isOpen,
+  });
+
+  return (
+    <div className={selectHeaderClasses} {...rest}>
```

Comment:
This div is not necessary, use `<></>` and put the logic of open, active, disabled at the parent

Reply:
Pour permettre de séparer ce qui est "Input" de ce qui est "Dropdown" au niveau du `Select` c'est bien pratique d'avoir un `<div>` qui englobe le tout.